soccer/src/soccer/robot_intent.hpp

```diff
@@ -13,7 +13,8 @@ struct RobotIntent {
 
     using Msg = rj_msgs::msg::RobotIntent;
     enum ShootMode { KICK, CHIP };
-    enum TriggerMode { STAND_DOWN, IMMEDIATE, ON_BREAK_BEAM };
+    enum TriggerMode { STAND_DOWN, IMMEDIATE, ON_BREAK_BEAM, AT_END };
```

Comment:
```suggestion
    enum TriggerMode { STAND_DOWN = 0, IMMEDIATE, ON_BREAK_BEAM, AT_END };
```
nit: technically you're not guaranteed it will start at 0

Reply:
wait actually? thats crazy